data/sql/updates/pending_db_world/rev_1588445740470602001.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588445740470602001');
+
+-- Ouro Spawner
+UPDATE `creature_template` SET `unit_flags` = 33685504, `ScriptName` = 'npc_ouro_spawner' WHERE (`entry` = 15957);
```

Comment:
for unit flags, please use `unit_flags`=`unitflags`|FLAG so you add the flags and not override the flags

Reply:
Okay, fixed